contracts/drafts/IERC1820Registry.sol

```diff
@@ -0,0 +1,90 @@
+pragma solidity ^0.5.2;
+
+/**
+ * @title ERC1820 Pseudo-introspection Registry Contract
+ * @author Jordi Baylina and Jacques Dafflon
+ * @notice For more details, see https://eips.ethereum.org/EIPS/eip-1820
+ */
+interface IERC1820Registry {
```

Comment:
How can users of OpenZeppelin get an instance of the registry as a variable?

Reply:
`IERC1820Registry registry = IERC1820Registry(0x1820b744B33945482C17Dc37218C01D858EBc714)`;

It'd be nice to provide a way to retrieve it, a sort of factory (that doesn't actually deploy it).

Reply:
Yeah that's what I meant. Okay let's tackle that in a different PR.